### ISSUE TITLE
downloading images reduces quality

### ISSUE THREAD

**Issue by sefibk** [PavelLaptev/JSON-to-Figma#63]
When I download an .svg file, it reduces the quality and does not keep it "vectorized".
Any way to keep the file in the svg format?

On the left of the image you can see a vectorized part while the right part of the image is the result of using your plugin:
![image](https://user-images.githubusercontent.com/38955896/216815835-c5a3ab12-afee-4372-bd4d-1f7e503ea2a8.png)

**Comment by PavelLaptev** [comment]
@sefibk correct. Right now I convert all images how I receive them in 1:1. But next version I'll do a check for the image format. If it's SVG I'll convert with 2x scale

**Comment by sefibk** [comment]
Thank you for the answer. 

1. Isn't there an option to inject it into Figma as a vectorized object (i.e. SVG?)

2. When can I expect the next version?

**Comment by PavelLaptev** [comment]
@sefibk 
1. Unfortunately, there is no way to use vector images as fills in Figma.
2. Probably only next month
